vfcompositor/Compositor.c

```
@@ -407,21 +413,64 @@ void VFCompositor(void) {
             MLFQYield();
         }
     }
-    WindowManagerInit();
+    CompositorInit(&g_compositor_ctx);
 
     while (1) {
         if (VBEIsInitialized()) {
             // Render text content for all windows that need it
-            Window* current = g_window_list_head;
+            if (HasInput()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)